arches/app/etl_modules/save.py

```diff
@@ -66,7 +66,7 @@ def save_to_tiles(userid, loadid, finalize_import=True, multiprocessing=True):
             }
         finally:
             try:
-                cursor.execute("""CALL __arches_complete_bulk_load();""")
+                cursor.execute("""CALL __arches_complete_bulk_load(%s);""",[finalize_import])
```

Comment:
I think we're trying to do too much in these functions (database as well as python functions). 
I would get rid of both `__arches_prepare_bulk_load` and `__arches_complete_bulk_load` altogether.  I would move the enabling and disabling of the triggers to the `save_to_tiles` function and call them directly. 

I would then create another function that encapsulates the sql below into another sql function.
```UPDATE resource_x_resource x
SET resourceinstancefrom_graphid = r.graphid
FROM resource_instances r
WHERE r.resourceinstanceid = x.resourceinstanceidfrom
AND x.resourceinstancefrom_graphid is null;
UPDATE resource_x_resource x
SET resourceinstanceto_graphid = r.graphid
FROM resource_instances r
WHERE r.resourceinstanceid = x.resourceinstanceidto
AND x.resourceinstanceto_graphid is null;```

I would then call that function after the call to `save_to_tiles` where needed.  Same goes for `__arches_refresh_spatial_views`.  That should be pulled out of the `save_to_tiles` method and used upstream as needed.

Reply:
It would be more explicit to call specific functions from `save_to_tiles`, but `__arches_prepare_bulk_load` and `__arches_complete_bulk_load` have been available since v6.2, and were introduced to support custom sql loads using spatial views. It seems like removing them completely should wait until v8 to maintain backward compatibility.